config/fedora.yml

```diff
@@ -0,0 +1,4 @@
+user: fedoraAdmin
```

Comment:
Uh nevermind, the ActiveFedora route would make this harder because it requires a 1-1 redeclaration of the original model